stats/docs/qdigest.md

```diff
@@ -0,0 +1,57 @@
+# Quantile Digest
+
+## Background
+
+The implementation of quantile digest in this library is based on 
+[q-digest [Shrivastava et al. ’04]](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.132.7343).
+It has been modified to support exponential decay based on 
+[Cormode et al. ’09](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.159.3978).
+
+## Format
+
+_Unless otherwise noted, all values are little-endian._
+
+### Quantile Digest Layout
+
+![QDigest Layout](qdigest_layout.png)
+
+The quantile digest algorithm uses a binary tree to store discrete ranges of values
+and their cardinality.  In this format, the nodes that make up the quantile digest's 
+binary tree are serialized as a post-order traversal of the tree.
+
+* format: byte value which represents the format used for serialization (currently
+just one format exists, `0`)
+* max error: represents the maximum error as a double
+* alpha: represents the exponential decay factor as a double
+* landmark: represents the seconds from epoch time used by the quantile
+digest's exponential decay algorithm as a long
+* min: the minimum value seen represented as a long, used to improve the 
+error on low quantile queries
+* max: the maximum value seen represented as a long, used to improve the 
+error on high quantile queries
+* \# of nodes - the number of nodes in this quantile digest represented as a long
```

Comment:
This probably doesn't need to be a long. An int should be sufficient.

Reply:
It was mistakenly referred to as a long here, in reality it is an int as you point out.